PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx

```diff
@@ -2360,7 +2380,7 @@ void AliAnalysisTaskGammaConvCalo::UserCreateOutputObjects(){
         fTrueJetHistograms[iCut]->Add(fHistoUnfoldingRejectInvMassZ[iCut]);
       }
 
-      if(!fDoLightOutput){
+      if(fDoLightOutput==0){
```

Comment:
these have to stay.

Reply:
Please also check that the dir gamma extraction will still work with this output level (ExtractSignalGammaV2.C).